### PR TITLE
Fix mobile fullwidth for inline MPU

### PR DESCRIPTION
**Pull request by Calum-Campbell**
## What does this change?
- ad-slot--container-inline containers will now be fullwidth when displayed on mobile.
## Does this affect other platforms - Amp, Apps, etc?

no
## Screenshots

Before:
image

After:
image
## Request for comment

@regiskuckaertz

### CODE REVIEW

**Comment by rich-nguyen** [comment]
👍

**Comment by Calum-Campbell** [comment]
@regiskuckaertz @rich-nguyen , asSlot now inserted with gutters not full screen width.

File: static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js [hunk 1]
```diff
@@ -106,6 +106,12 @@ define([
         }
     }
 
+    function setFluid(el) {
+        if (el.classList.contains('ad-slot--container-inline')) {
+            el.classList.add('ad-slot__fluid250');
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js]: Have you tried with `ad-slot--fluid`? the `ad-slot__fluid250` has lots of side-effects which may be problematic in the long term.

**Comment by regiskuckaertz** [comment]
Yep, I think that's all good, just the question. 👍

File: static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js (hunk 1 above)
Comment (Calum-Campbell) [inline static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js]: Ah yes, was just messing around this morning, forgot to change back, thanks!

**Pull request merged**